diff --git a/reflex-dom-core.cabal b/reflex-dom-core.cabal
index 0da6932..d8c4517 100644
--- a/reflex-dom-core.cabal
+++ b/reflex-dom-core.cabal
@@ -53,7 +53,7 @@ library
     contravariant >= 1.4 && < 1.5,
     data-default >= 0.5 && < 0.8,
     dependent-map == 0.2.*,
-    dependent-sum >= 0.3 && < 0.5,
+    dependent-sum >= 0.3 && < 0.6,
     dependent-sum-template >= 0.0.0.4 && < 0.1,
     directory >= 1.2 && < 1.4,
     exception-transformers == 0.4.*,
@@ -133,7 +133,7 @@ library
 
   if flag(use-template-haskell)
     build-depends:
-      dependent-sum >= 0.3 && < 0.5,
+      dependent-sum >= 0.3 && < 0.6,
       dependent-sum-template >= 0.0.0.5 && < 0.1,
       template-haskell
     other-extensions: TemplateHaskell
@@ -142,7 +142,7 @@ library
       Reflex.Dom.Builder.Class.TH
   else
     build-depends:
-      dependent-sum == 0.4.*
+      dependent-sum >= 0.3 && < 0.6
 
 test-suite hlint
   build-depends: base, hlint
